molecule/builder-xenial/tests/test_security_updates.py

```diff
@@ -1,16 +1,31 @@
 import os
+from subprocess import check_output
+import re
+import pytest
+
 SECUREDROP_TARGET_PLATFORM = os.environ.get("SECUREDROP_TARGET_PLATFORM")
 testinfra_hosts = [
         "docker://{}-sd-sec-update".format(SECUREDROP_TARGET_PLATFORM)
 ]
 
 
+def is_rc_version():
+    command = "git describe --all"
+    version = check_output(command, shell=True).\
+        decode("utf8")[0:-1]
+    result = re.match(r"(^[\d]+\.[\d]+\.[\d]+-rc[\d]+)|(^heads/update-builder.*)", version)
```

Comment:
it seems like this RE will not match any release tags/branches

Reply:
updated for tags/u.v.w-rcX, heads/release/u.v.w, and heads/update-builder* cases